frontend/src/pages/NewRunV2.tsx

```diff
@@ -440,7 +505,47 @@ function NewRunV2(props: NewRunV2Props) {
         {/* One-off/Recurring Run Type */}
         {/* TODO(zijianjoy): Support Recurring Run */}
         <div className={commonCss.header}>Run Type</div>
-        <div>Only one-off run is supported for KFPv2 Pipeline at the moment.</div>
+        {/* <div>Only one-off run is supported for KFPv2 Pipeline at the moment.</div> */}
+        {isClone && <span>{isRecurringRun ? 'Recurring' : 'One-off'}</span>}
```

Comment:
How is `isClone` determined? I think we need to read from query parameter `cloneFromXXX`

Reply:
Correct, we should put the `isClone` in the future design.